app.rb

```diff
@@ -3,18 +3,77 @@
 tenants = data[:tenants]
 
 # First, Open the data.rb an inspect the data. Identify and write, in comments, the following:
-  # Explain how the data is structured
-  # What are the properties for each of the two types of hashes
+             # Explain how the data is structured
+#  Ans
+             # What are the properties for each of the two types of hashes
+#
+
 
 # Use enumerables to -
   # Print all the addresses for the apartments
+  module Enumerable
+    def pluck(key)
+      map {|obj| obj[key] }
+    end
+  end
+
+  apartments.pluck(:address)
+
+  #or
+
+  apartments.each do |apartment|
+         puts apartment[:address]
+       end
   # Print all the names for tenants
+  tenants = data[:tenants]
+
+tenants.each do |tenant|
+puts tenant[:name]
+end
   # Print only apartments that are less then 700 in rent
+
+  apartments.each do |apartment|
+  while apartment[:monthly_rent] < 700
```

Comment:
It would make more sense to use an `if` statement here since checking the condition once in each iteration of the enumerable.